### PR TITLE
Localization via resx #17

### PR DESCRIPTION
**Pull request by makeProjectGreatAgain**
Added localization
Made buttons a little bigger
Tried not to change anything else

### CODE REVIEW

**Comment by PandaWood** [comment]
Thanks looks good, I'll try to run it up and give it a quick test shortly

**Comment by makeProjectGreatAgain** [comment]
I also added this test, you can use it

```
                [Test]
		[Ignore("UI")]
		public static void TestRunManually()
		{
			var thread = new Thread(() =>
			{
			  Thread.CurrentThread.CurrentCulture = CultureInfo.GetCultureInfoByIetfLanguageTag("ru");
			  Thread.CurrentThread.CurrentUICulture = CultureInfo.GetCultureInfoByIetfLanguageTag("ru");
			  var er = new ExceptionReporter
				{
					Config =
					{
						AppName = "PhotoFuzz",
						AppVersion = "1.0",
						CompanyName = "photofuzz",
						SendMethod = ReportSendMethod.SimpleMAPI,
						EmailReportAddress = "PhotoFuzz@gmail.com",
						ShowLessDetailButton = true,
						TakeScreenshot = true,
						ReportTemplateFormat = TemplateFormat.Markdown,
						ReportCustomTemplate = "Done!",
						AttachmentFilename = $"{DateTime.UtcNow.ToString("dd-MM-yy_HH-mm")}_report",
						FilesToAttach = new[] {"app.log"}
					}
				};
				var ex = new Exception("Test Exception");
				er.Show(ex);
			});
			thread.SetApartmentState(ApartmentState.STA);
			thread.Start();
			thread.Join();
		}
```

**Comment by PandaWood** [comment]
I noticed the whole thing doesn't work (including test) if you open the solution `ExceptionReport.NET-traditional.sln`
So we might have to ditch that one at the same time - if that's reasonable...

File: src/Tests/ExceptionReporter_Tests.cs [hunk 1]
```diff
@@ -69,7 +71,46 @@ public void Can_Init_Config_Using_Object_Initializer()
 					FilesToAttach = new[] {"app.log"}
 				}
 			};
+
 			er.Show();
 		}
+
+		[Test]
+		[Ignore("UI")]
+		public static void TestRunManually()
```
Comment (PandaWood) [inline src/Tests/ExceptionReporter_Tests.cs]: We might have to call this `ManualLocalizationTest` and might be best in it's own file eg `ExceptionReporter_ManualTests.cs`

File: src/ExceptionReporting/ExceptionReporter.NET-sdk.csproj [hunk 2]
```diff
@@ -33,6 +33,25 @@
     <Reference Include="System.Management" />
     <Reference Include="System.Windows.Forms" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (PandaWood) [inline src/ExceptionReporting/ExceptionReporter.NET-sdk.csproj:36]: So I suppose we don't want to repeat doing this in `ExceptionReporter.NET-traditional.sln` so we better delete that old solution - I think VS2019 or greater is required to read the new project style (?) ... something like that.

**Review by PandaWood — CHANGES_REQUESTED** [review]
I think just these 2 comments need action before we finalize it
I'm not sure how many people will be affected by the "traditional" solution being deleted, but maybe the cord needs to be cut and avoid managing 2 solutions.

File: src/ExceptionReporting/ExceptionReporter.NET-sdk.csproj (hunk 2 above)
Comment (makeProjectGreatAgain) [reply src/ExceptionReporting/ExceptionReporter.NET-sdk.csproj:36]: >  I think VS2019 or greater is required to read the new project style (?)

VS2019 supports both project style.

> So I suppose we don't want to repeat doing this in ExceptionReporter.NET-traditional.sln so we better delete that old solution

I vote to delete ExceptionReporter.NET-traditional.sln, but you decide

File: src/Tests/ExceptionReporter_Tests.cs (hunk 1 above)
Comment (makeProjectGreatAgain) [reply src/Tests/ExceptionReporter_Tests.cs]: > We might have to call this `ManualLocalizationTest` and might be best in it's own file eg `ExceptionReporter_ManualTests.cs`

Yes, I agree with you.

Should I add two tests `ManualLocalizationRuTest` and `ManualLocalizationEnTest`?

Comment (makeProjectGreatAgain) [reply src/Tests/ExceptionReporter_Tests.cs]: > We might have to call this ManualLocalizationTest and might be best in it's own file eg ExceptionReporter_ManualTests.cs

Done

File: src/ExceptionReporting/ExceptionReporter.NET-sdk.csproj (hunk 2 above)
Comment (PandaWood) [reply src/ExceptionReporting/ExceptionReporter.NET-sdk.csproj:36]: Yeah thanks, I might do that in another commit after the merge and let this go through

**Pull request merged**